src/main/java/transfer/countrytrans/CountryTransfer.java

```diff
@@ -0,0 +1,30 @@
+package transfer.countrytrans;
+
+import io.restassured.response.Response;
+import transfer.statetrans.StateTransfer;
+
+import static io.restassured.RestAssured.given;
+
```

Comment:
CountryTransfer shouldn`t be extended from StateTransfer. It`s 2 separated endpoints

Reply:
changed